client/my-sites/site-settings/disconnect-site/pagination-flow.jsx

```diff
@@ -0,0 +1,53 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import page from 'page';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Gridicon from 'gridicons';
+import { getSelectedSiteSlug } from 'state/ui/selectors';
+
+class PaginationFlow extends Component {
+	verifySiteIsJetpack() {
+		if ( this.props.siteIsJetpack === false ) {
+			this.redirectToGeneral();
+		}
+	}
+
+	redirectToGeneral = () => {
+		const { siteSlug } = this.props;
+		page( '/settings/general/' + siteSlug );
+	};
+
+	clickHandler = () => {
+		this.redirectToGeneral();
+	};
+
+	render() {
+		const { translate } = this.props;
+		return (
+			<div className="disconnect-site__pagination main">
+				<Button
+					compact
+					borderless
+					className="disconnect-site__pagination skip"
```

Comment:
Do we need the `.skip` class?

Reply:
True, I don't think we need ```className``` here at all (at least for now).